### PR TITLE
This fixes funny error on envs substractions

### PR DESCRIPTION
**Pull request by santiagorodsor**

### CODE REVIEW

File: environment_kernels/core.py [hunk 1]
```diff
@@ -166,7 +166,7 @@ def _get_env_data(self, reload=False):
             env_data.update(supplyer(self))
 
         env_data = {name: env_data[name] for name in env_data if self.validate_env(name)}
-        new_kernels = env_data.keys() - self._env_data_cache.keys()
+        new_kernels = [env for env in list(env_data.keys()) if env not in list(self._env_data_cache.keys())]
```
Comment (janschulz) [inline environment_kernels/core.py]: Ah, the joy of supporting py2.7 :-)

I would use `set(env_data.keys()) - set(self._env_data_cache.keys())`, it's a bit shorter, but the above looks also fine.

**Comment by santiagorodsor** [comment]
"Only those with write access to this repository can merge pull requests."

**Comment by gautamkmr** [comment]
When can be this available in releasE ?

**Comment by Cadair** [comment]
Sorry I haven't got to baking a release yet, I will try and do it today.

**Comment by Cadair** [comment]
I just pushed 1.1.1 to pypi with this and #32 included. Sorry it took me so long.

**Pull request merged**